### PR TITLE
chore: Automate issues and prs addition to project

### PR DESCRIPTION
**Pull request by veekeys**
Github released this new action
https://github.com/actions/add-to-project

The concern I have is the token, not sure, if its gonna work.

UPDATE: ok, I am sure it wont work, cause its limited per repo.. We need token with the access to the org. 

LAST UPDATE: for now, I have linked my personal access token. I have tested with this PR and it successfully added it to project! I assume it would work with issues too.
For future, we might want to replace my token with some general org one, so thats gonna happen just maybe not that fast.

### CODE REVIEW

**Comment by mayank99** [comment]
Maybe you can create a PAT?
https://github.com/actions/add-to-project#creating-a-pat-and-adding-it-to-your-repository

**Comment by veekeys** [comment]
> Maybe you can create a PAT? [actions/add-to-project#creating-a-pat-and-adding-it-to-your-repository](https://github.com/actions/add-to-project#creating-a-pat-and-adding-it-to-your-repository)

I can, but not with org scope :)

**Comment by mayank99** [comment]
> I can, but not with org scope :)

Are you sure?
image

File: .github/workflows/add-issues-and-prs-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name: Add issues and pull requests to project
```
Comment (mayank99) [inline .github/workflows/add-issues-and-prs-to-project.yml:1]: Looks weird right now
image


Maybe make this name shorter and add a name to the job.

Other ones look cleaner for example
image

File: .github/workflows/add-issues-and-prs-to-project.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    types:
+      - opened
+
+jobs:
+  add-to-project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
+        with:
+          project-url: https://github.com/orgs/iTwin/projects/3/views/1
```
Comment (mayank99) [inline .github/workflows/add-issues-and-prs-to-project.yml:17]: I think there's no need for views here
```suggestion
          project-url: https://github.com/orgs/iTwin/projects/3
```

File: .github/workflows/add-issues-and-prs-to-project.yml (hunk 1 above)
Comment (veekeys) [reply .github/workflows/add-issues-and-prs-to-project.yml]: added

File: .github/workflows/add-issues-and-prs-to-project.yml (hunk 2 above)
Comment (veekeys) [reply .github/workflows/add-issues-and-prs-to-project.yml]: good catch, changed

**Review by mayank99 — APPROVED** [review]

**Pull request merged**